ios/RNViewShot.m

```diff
@@ -88,7 +101,17 @@ - (dispatch_queue_t)methodQueue
       scrollView.contentOffset = CGPointZero;
       scrollView.frame = CGRectMake(0, 0, scrollView.contentSize.width, scrollView.contentSize.height);
     }
-    UIGraphicsBeginImageContextWithOptions(size, NO, 0);
+
+    if (target == -1) {
```

Comment:
this block also have issues

<img width="767" alt="screen shot 2017-09-16 at 09 35 03" src="https://user-images.githubusercontent.com/211411/30510363-83600a94-9ac2-11e7-95ac-80f07d8b6c1a.png">

Reply:
Error resolved.